src/locales/ge/privacy-policy.json

```diff
@@ -0,0 +1,59 @@
+        "content": "Diese Datenschutzrichtlinie kann nach alleinigem Ermessen des Unternehmens geändert werden. Wenn wesentliche Änderungen vorgenommen werden, wird der Nutzer in dem von ihm genutzten Dienst benachrichtigt. Beachten Sie, dass das Hinzufügen neuer Produkte, die unter den Begriff Software fallen, z. B. ein neues Produkt für den Benutzer, das die App ersetzt, oder ein neues Tool zum Hochladen von Inhalten, nicht als wesentlich angesehen wird, da es sich nicht auf die Benutzer auswirkt, es sei denn, sie übernehmen das neue Produkt. Die Änderung von Softwarenamen, der in dieser Datenschutzrichtlinie verwendeten Terminologie und der Änderung von Linkpositionen sind ebenfalls Beispiele für nicht wesentliche Änderungen."
+      },
+      "informationCollected": {
+        "title": "3. gesammelte Informationen",
+        "content": "Alle Daten, die in die Blockchain geschrieben werden, werden implizit nicht nur vom Unternehmen gesammelt, sondern auch von jedem anderen auf der Welt, der den Full Node lokal ausführt oder auf den über die App oder eine dritte Partei zugegriffen wird. Dazu gehören unter anderem Inhalts-Hashes, Mitgliedschaftsprofile, Memo-Felder und jede andere Art und Weise, wie ein Nutzer Daten in der Blockchain aufzeichnen kann. <0/><0/> Bei der Nutzung der <1>Faucet</1> Unterseite der Website zeichnet das Unternehmen die IP-Adresse auf, die hinter jeder neuen Anfrage nach Token steht. Diese Daten werden alle 14 Tage gelöscht."
+      },
+      "googleAnalytics": {
+        "title": "",
+        "content": "Das Unternehmen verwendet <0>Google Analytics</0>, mit IP-Anonymisierung, um Statistiken über die Website und die von uns gehostete Version der App zu sammeln. Alle anpassbaren Einstellungen für die gemeinsame Nutzung von Daten sind ausgeschaltet, um die Privatsphäre der Nutzer zu schützen. <1 /><1 /> <2>Das Unternehmen verkauft Ihre Daten nicht für Werbe- oder andere Zwecke.</2>"
+      }
+    },
+    "cookiePolicyItems": {
+      "title": "Cookie-Richtlinie",
+      "subtitle": "Zuletzt aktualisiert am 17. April 2019",
+      "whatAreCookies": {
+        "title": "1. Was sind Cookies?",
+        "content": "Cookies sind kleine Textdateien, die von einer von Ihnen besuchten Website an Ihren Webbrowser gesendet werden. Eine Cookie-Datei wird in Ihrem Webbrowser gespeichert und ermöglicht es dem Dienst oder einem Dritten, Sie zu erkennen und Ihren nächsten Besuch zu erleichtern und den Dienst für Sie nützlicher zu machen. <0 /><0 /> Cookies können dauerhafte oder Sitzungs-Cookies sein."
+      },
+      "howWeUseCookies": {
+        "title": "2. Wie wir Cookies verwenden",
+        "content": "Wir verwenden Cookies für die folgenden Zwecke unseren Service: <0 /><0 /><1><0>Analytik bereitstellen</0><0>Einstellungen speichern</0><0>Lokale Speicherung von Schlüsseln und Mitgliedschaften.</0></1>"
+      },
+      "thirdPartyCookies": {
+        "title": "3. Third-Party-Cookies",
```

Comment:
Drittparteien

Reply:
How about Dritte Party Kekse? Kidding. Changed it. do Drittparteien-cookies.